ui/packages/image/src/Sketch.svelte

```diff
@@ -52,7 +52,14 @@
 					ctx.temp.drawImage(value_img, 0, 0);
 					ctx.temp.restore();
 				} else {
-					ctx.temp.drawImage(value_img, 0, 0);
+					const image = draw_cropped_image();
+					ctx.temp.drawImage(
+						value_img,
+						image.x,
+						image.y,
+						image.width,
+						image.height
+					);
```

Comment:
Might be a silly question, but do we still need this? I thought the last line of `draw_cropped_image()` does this

Reply:
wait, did i leave one in?

Reply:
i'll check

Reply:
okay i did, it is gone now.